README.md

```diff
@@ -10,11 +10,11 @@
 
 Branch | Travis | Coveralls | Scrutinizer |
 ------ | ------ | --------- | ----------- |
-2.x   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
+3.0-dev   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
```

Comment:
this is weird, as -dev does not sound stable - and you now have twice 3.0-dev...